celer/tests/test_mtl.py

```diff
@@ -57,9 +58,11 @@ def test_GroupLasso_MultitaskLasso_equivalence():
     X_data = np.empty([1], dtype=X.dtype)
     X_indices = np.empty([1], dtype=np.int32)
     X_indptr = np.empty([1], dtype=np.int32)
+    weights = np.ones(len(grp_ptr))
```

Comment:
this has 1 too many elements

Reply:
should I decrease the size of weights by 1?

Reply:
yes, grp_ptr has shape (n_groups + 1,)